src/common/cockpitsystem.c

```diff
@@ -215,3 +217,24 @@ cockpit_system_process_start_time (void)
 
   return start_time;
 }
+
+char *
+cockpit_system_session_id (void)
+{
+  char *session_id;
+  pid_t pid;
+  int res;
+
+  pid = getppid ();
```

Comment:
It's not really clear why this uses the parent PID (cockpit-session) instead of the bridge's own pid. However, this is already the case in the current code, it just caught my eye.